components/style/properties/longhand/box.mako.rs

```diff
@@ -1624,40 +1551,160 @@ ${helpers.predefined_type("scroll-snap-coordinate",
 
         #[inline]
         fn to_computed_value(&self, context: &Context) -> computed_value::T {
+            use self::SpecifiedOperation::*;
             if self.0.is_empty() {
                 return computed_value::T(None)
             }
 
             let mut result = vec!();
             for operation in &self.0 {
                 match *operation {
-                    SpecifiedOperation::Matrix(ref matrix) => {
-                        result.push(computed_value::ComputedOperation::Matrix(matrix.to_computed_value(context)));
+                    Matrix { a, b, c, d, e, f } => {
+                        let mut comp = computed_value::ComputedMatrix::identity();
+                        comp.m11 = a.to_computed_value(context);
+                        comp.m12 = c.to_computed_value(context);
```

Comment:
the bug is here: `comp.m12` should be `b`, not `c`. Similarly, `comp.m21` should be `c`, not `b`.